src/mozilla_linux_pkg_manager/cli.py

```diff
@@ -331,35 +209,25 @@ def main():
     clean_up_parser.add_argument(
         "--retention-days",
         type=int,
-        help="Retention period in days for packages in the nightly channel",
+        required=True,
+        help="Retention period in days for the selected packages",
     )
     clean_up_parser.add_argument(
         "--dry-run",
         action="store_true",
         help="Do a no-op run and print out a summary of the operations that will be executed",
         default=False,
     )
+    clean_up_parser.add_argument(
```

Comment:
What is the benefit of `skip-delete` vs. running in `dry-run` mode? It looks to me like the latter is a more extensive test, at perhaps the cost of some extra network costs?

Reply:
The difference is`dry-run` will fail is the SDK doesn't have delete permissions on the resource.

Reply:
Ah - I see. So this allows for running a dry run with a lower permissioned account - makes sense!